### DISCUSSION TITLE
Installation problems masterlist

### DISCUSSION THREAD

**Discussion by Pinsplash** [Pinsplash/halflife2chaos#121] [General]
Quite a few different installation problems have come up over time. Make sure you:
- Downloaded the correct thing. You want [hl2c.zip](https://github.com/Pinsplash/halflife2chaos/releases), not the source code. This is the problem 99% of the time.
- Have switched Source SDK Base 2013 Singleplayer to the "upcoming - upcoming" beta.
- Restart Steam after installing.
- Have both Half-Life 2 and the mod installed to the same Steam directory (i.e. the steamapps folder). If you enter the hl2chaos folder, go 2 folders up, and go into common, Half-Life 2 should be there.
- Are opening the mod and not base HL2. The mod will actually be called "HL2 Chaos" in Steam.
- Don't put the hl2c folder in the sourcemods folder. The hl2chaos, ep1chaos, and ep2chaos folders should be put directly in the sourcemods folder.
- Check that Steam didn't put the mod in your hidden games. (#77)

**Comment by none-what** [comment]
:)
